src/v2/cookbook/serverless-blog.md

```diff
@@ -115,18 +113,17 @@ Then create `components/BlogHome.vue` which will be your blog homepage that list
     }
   }
 </script>
-Display the result
 
 <template>
   <div id="blog-home">
       <h1>{{ page_title }}</h1>
-      <!-- Create v-for and apply a key for Vue. Example is using a combination of the slug and index -->
+      <!-- Create `v-for` and apply a `key` for Vue. Here is using a combination of the slug and index. -->
```

Comment:
Here is using isn't grammatically correct. It should either be "This example is..." or "Here we are using"

Reply:
Done for this. 👌